src/promptflow-evals/tests/evals/unittests/test_save_eval.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, List, Optional
+
+import inspect
+import os
+import tempfile
+import unittest
+from promptflow.client import PFClient
+from promptflow.evals import evaluators
+from promptflow.evals.evaluators import content_safety
+
+
+class TestSaveEval(unittest.TestCase):

```

Comment:
Please switch to pytest and add @pytest.mark.unittest mark to the class otherwise I thnk it will be ignored by the CI pipeline.

Reply:
Use test_chat_evaluator.py as example

Reply:
Fixed.